### PR TITLE
Login page router

### PR DESCRIPTION
**Pull request by AleksandarGT**

### CODE REVIEW

**Review by MolotovHammr — APPROVED** [review]

File: src/components/Sensor/sensor.css [hunk 1]
```diff
@@ -25,7 +25,7 @@
   border: solid;
   border-color: #009ddc;
 
-  filter: drop-shadow(0px 3px 4px rgba(0, 0, 0, 0.25));
+  filter: drop-shadow(0px 3px 1px rgba(0, 0, 0, 0.25));
```
Comment (BondarencoM) [inline src/components/Sensor/sensor.css]: I think you forgot to reverse this one

File: src/index.js [hunk 2]
```diff
@@ -14,7 +15,9 @@ const root = document.getElementById('root');
 ReactDOM.render(
   <React.StrictMode>
     <ApolloProvider client={client}>
-      <App />
```
Comment (BondarencoM) [inline src/index.js:17]: I think you should put the strict mode back

**Review by BondarencoM — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (AleksandarGT) [reply src/index.js:17]: Isn't strict mode there? At least on my screen it is lol

Comment (BondarencoM) [reply src/index.js:17]: did I tell you I'm blind -_- ? 
Yeah, that's great

**Review by BondarencoM — APPROVED** [review]

**Review by oererenn — APPROVED** [review]

**Review by michaelbahchevanov — APPROVED** [review]

**Review by larsverp — APPROVED** [review]
I don't really like the login button, but that's just personal taste. The rest of then page is looking good 🎉

**Pull request merged**